### DISCUSSION TITLE
Question on  "configurating a New robot"

### DISCUSSION THREAD

**Discussion by zhangsanfeng86** [NVlabs/curobo#18] [Software Q&A]
Hi,

I follow the Tutorial with a UR5e robot. 
When run "omni_python curobo/examples/isaac_sim/convert_urdf_to_usd.py --robot ur5e.yml --save_usd",
But no window popup with isaac sim.

<img width="1378" alt="image" src="https://github.com/NVlabs/curobo/assets/73547638/1dae25b1-5016-45af-8457-cc34f080cdd0">

**Comment by balakumar-s** [comment]
That script will run headless and only convert the urdf to usd. It should have created a `.usd` file.

**Reply by zhangsanfeng86** [reply]
OK.  I understand

**Reply by zhangsanfeng86** [reply]
Could you give some discription on this items or how get the docment ?

<img width="773" alt="image" src="https://github.com/NVlabs/curobo/assets/73547638/376599cb-3b91-4620-b83c-5c12ebb8798b">

**Reply by zhangsanfeng86** [reply]
I get confuse:  how to get the value:
<img width="737" alt="image" src="https://github.com/NVlabs/curobo/assets/73547638/67036f8b-2618-498d-843f-5fe8f4c4c08b">

**Reply by balakumar-s** [reply]
Joint names: They come from the robot urdf
retract_cfg: They are a home configuration for the robot
weights can be 1 for all joints.
The max_jerk and max_acceleration comes from the robot specifications.

Is there a specific robot that you are trying to build the configuration for? We have made configurations for some robots here: https://curobo.org/source/notes/01_robot_list.html

**Reply by zhangsanfeng86** [reply]
Thank you very much for your helpful answer!
